OpenRA.Game/Map/MapPreview.cs

```diff
@@ -418,7 +418,7 @@ public void UpdateRemoteSearch(MapStatus status, MiniYaml yaml, Action<MapPrevie
 
 		public void Install(string mapRepositoryUrl, Action onSuccess)
 		{
-			if (Status != MapStatus.DownloadAvailable || !Game.Settings.Game.AllowDownloading)
+			if (Status != MapStatus.DownloadError && (Status != MapStatus.DownloadAvailable || !Game.Settings.Game.AllowDownloading))
```

Comment:
This should be `((Status != MapStatus.DownloadError && Status != MapStatus.DownloadAvailable) || !Game.Settings.Game.AllowDownloading)`. We want to unconditionally exit here if the player has disabled map downloading.

Reply:
Makes sense